base/commands/migration/migration_stages.go

```diff
@@ -269,6 +269,22 @@ func finalizeMigration(ctx context.Context, ec plug.ExecContext, ci *hazelcast.C
 	return nil
 }
 
+func maybePrintWarnings(ctx context.Context, ec plug.ExecContext, ci *hazelcast.ClientInternal, migrationID string) {
+	q := fmt.Sprintf(`SELECT JSON_QUERY(this, '$.warnings' WITH WRAPPER) FROM %s WHERE __key='%s'`, StatusMapName, migrationID)
+	row, err := querySingleRow(ctx, ci, q)
+	if err != nil {
+		ec.Logger().Error(err)
+		return
+	}
+	var warnings []string
+	err = json.Unmarshal(row.(serialization.JSON), &warnings)
+	if err != nil {
+		ec.Logger().Error(err)
+		return
+	}
+	ec.PrintlnUnnecessary("* " + strings.Join(warnings, "\n* "))
```

Comment:
If there're lots of them, it can be more reasonable to point to the migration report instead. The warnings will be in the report as well. If there're less than 5 we can directly print them

Reply:
I think there won't be too many warning that makes it hard to display in the terminal.

Reply:
There can be in the future. We'll keep adding things to warnings field.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/420/commits/64e39bc69b8d0d2ef7548d97ea3f9308c64ab314

Reply:
Note that, it's not just for future. 1000 ds skips can even happen now.